packages/hub/searchers.js

```diff
@@ -215,6 +241,7 @@ class Searchers {
       upstreamDoc,
       includePaths,
       routers: this._getRouters(),
+      relatedLinks: this._getRelatedLinks.bind(this),
```

Comment:
based on my earlier comment, I would not pass this function into `DocumentContext` until we can support invalidation of models that appear in the query based relationships. but rather materialize these queries after getting the pristine doc from the `DocumentContext.

Reply:
👍